### PR TITLE
Adding Watchguard FireboxV all version

### PR DESCRIPTION
**Pull request by Acronychal**
Tested on CML 2.6 & 2.7

### CODE REVIEW

**Comment by Acronychal** [comment]
Watchguard FireboxV node definition pull request

**Comment by Acronychal** [comment]
They do have a dimension server that runs off a .ova/qcow2. I can test and integrate that too. I've not had the need to use it yet.

**Comment by xorrkaz** [comment]
You don't have to integrate if you don't want to, but it might make sense to put this in its own subdirectory

**Comment by Acronychal** [comment]
I agree.

File: node-definitions/watchguard/README.md
Comment (xorrkaz) [inline node-definitions/watchguard/README.md:1]: Can you change this into an overall Watchguard readme, and then move this file to the fireboxv subdir?

**Review by xorrkaz — CHANGES_REQUESTED** [review]

File: node-definitions/watchguard/README.md
Comment (Acronychal) [reply node-definitions/watchguard/README.md:1]: Done.

Comment (xorrkaz) [reply node-definitions/watchguard/README.md:1]: Looks good to me!

**Review by xorrkaz — APPROVED** [review]

**Pull request merged**